packages/shared/src/components/tooltips/BaseTooltipContainer.tsx

```diff
@@ -62,7 +62,11 @@ export function BaseTooltipContainer({
       {!!showArrow && (
         <Arrow
           data-popper-arrow
-          className={classNames(styles.tippyTooltipArrow, bgClassName)}
+          className={classNames(
+            styles.tippyTooltipArrow,
+            bgClassName,
+            'text-justify',
```

Comment:
This ensures the placement of the arrow won't be affected by inherited classes. Should not break existing tooltips. Preview after this was applied:
![Screenshot 2024-11-30 at 8 44 26 PM](https://github.com/user-attachments/assets/bb978c6d-95f6-40f8-9844-79e5637c7477)
![Screenshot 2024-11-30 at 8 44 22 PM](https://github.com/user-attachments/assets/92d20031-4ff9-44fd-978a-2eca9de8753a)
![Screenshot 2024-11-30 at 8 44 20 PM](https://github.com/user-attachments/assets/647cb990-dae2-4836-838c-82d0f63d2f06)